vpr/src/noc/odd_even_routing.h

```diff
@@ -0,0 +1,27 @@
+#ifndef VTR_ODD_EVEN_ROUTING_H
+#define VTR_ODD_EVEN_ROUTING_H
+
+#include "turn_model_routing.h"
+
```

Comment:
Probably comment what is this class for ... what is the odd/even routing algorithm.